conf/battle/items.conf

```diff
@@ -121,3 +121,7 @@ allow_bound_sell: 0x0
 // no = normal refine chances in effect (official/default value)
 // yes = event refine chances in effect
 event_refine_chance: no
+
+// Hide n last characters of player's name with asterisk (*) when the player
+// obtained an item with special broadcast flag.
```

Comment:
Documentation for names longer than configured value is missing.

Reply:
eh, how I should write it?

Reply:
If you set this configuration too high the full name of players with names shorter than the specified value will get censored completely.
Something like this. Maybe @aleos89 can give us input as well. :-)

Reply:
`Note: Players with short names can be fully converted to asterisks if this config value is set high.`